scripts/sync-workflow.js

```diff
@@ -58,3 +61,59 @@ function login(resolve, reject) {
     return resolve(resBody);
   });
 }
+
+function emptySync(loginResult) {
+  return function(resolve, reject) {
+    lr.actStart('EMPTY SYNC');
+
+    const myRequest = request.defaults({
+      headers: requestBodyUtils.getSyncRequestHeaders({
+        'X-FH-cuid': clientIdentifier,
+        'X-FH-sessionToken': loginResult.sessionToken
+      })
+    });
+
+    const reqBody = requestBodyUtils.getSyncRequestBody({
+      dataset_id: 'workorders',
+      query_params: {
+        "filter": {
+          "key": "assignee",
+          "value": "rkX1fdSH"
+        }
+      },
+      dataset_hash: '833c31caee5b87f4cbf9baeab0f5dce6814467ee',
+      meta_data: {
```

Comment:
This bit I copied from @aidenkeating. I think in the browser requests that I was looking at, `meta_data` was set to `null`. I've tried both ways, and it doesn't seem to have an effect on the result.